$MirOS$

	He says this file does not contain fixed sizes... well, it did.

--- src/selfnosize.css.orig	Wed Mar 23 22:31:41 2005
+++ src/selfnosize.css	Sun Mar 27 17:22:22 2005
@@ -18,7 +18,7 @@ h2 { margin-bottom:18px; }
 
 li { margin-top:3px; }
 
-pre { font-family:"Courier New",Courier; font-size:13.4px; color:#0000C0; background-color:#FFFFE0; border:10px solid #EEEEEE; padding:10px; white-space:pre; }
+pre { font-family:"Courier New",Courier; color:#0000C0; background-color:#FFFFE0; border:10px solid #EEEEEE; padding:10px; white-space:pre; }
 
 td pre {border-style:none; padding:0;}
 
